From f54075708b546bfc4334d831183cf9557d0556be Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@freescale.com>
Date: Wed, 1 Apr 2015 21:36:55 -0500
Subject: [PATCH 027/132] BRINGUP ONLY: Don't set baud rate

Temporary workaround for bringup to rely on U-Boot setting the baud
rate.

Code was copied from Scott's patch in the 3.16 bringup branch

Signed-off-by: Stuart Yoder <stuart.yoder@freescale.com>
[Original patch from FSL LS2085 SDK EAR4.0]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/tty/serial/8250/8250_core.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/tty/serial/8250/8250_core.c b/drivers/tty/serial/8250/8250_core.c
index dfc63d9..9a5610f 100644
--- a/drivers/tty/serial/8250/8250_core.c
+++ b/drivers/tty/serial/8250/8250_core.c
@@ -2444,7 +2444,10 @@ serial8250_do_set_termios(struct uart_port *port, struct ktermios *termios,
 	else
 		serial_port_out(port, UART_LCR, cval | UART_LCR_DLAB);
 
+#if !defined(CONFIG_ARCH_FSL_LS2085A) \
+       && !defined(CONFIG_ARCH_FSL_LS2085A_MINIMAL)
 	serial_dl_write(up, quot);
+#endif
 
 	/*
 	 * XR17V35x UARTs have an extra fractional divisor register (DLD)
-- 
1.7.5.4

